src/target/riscv/riscv-013.c

```diff
@@ -1974,7 +1974,7 @@ static int examine(struct target *target)
 				info->impebreak);
 	}
 
-	if (info->progbufsize < 4 && riscv_enable_virtual) {
```

Comment:
The program buffer size check `info->progbufsize < 4` does not look right because it does not account for `impebreak`. Can you please check it.

I believe `has_sufficient_progbuf()` should be used instead.

Reply:
I will address this in future MR

Reply:
Ok, thank you.